Enhanced the hero section  (issue #2)

Prachiikhar25 opened the pull request:
✅ Added a badge/tag labeled "🤖 AI-Powered Fitness Assistant" - positioned at the top with a teal color scheme that matches the site design

✅ Added call-to-action buttons - "Try the Chatbot" (primary button linking to #features) and "View Setup Guide" (secondary button linking to #setup) with hover effects and icons

✅ Improved spacing and visual hierarchy - added mb-8 to the description text and organized the layout with proper spacing between elements

The changes maintain the existing design language while making the hero section more engaging and actionable for users.
<img width="1880" height="874" alt="image" src="https://github.com/user-attachments/assets/f5d099a4-ac20-4c04-a200-f6f6d9c4b331" />

Prachiikhar25 committed:
Enhanced the hero section

Akki-jaiswal reviewed (changes requested):
Thanks @Prachiikhar25 for your contribution..you have made some significant changes. I want that if we can add a functional chatbot button which is often present at almost every web app.

Prachiikhar25 committed:
enhanced the hero section

Prachiikhar25 committed:
Resolved the merge conflict

Prachiikhar25 committed:
Merge branch 'main' into feat

Prachiikhar25 commented:
@Akki-jaiswal  I have resolved the merge conflicts you can merge this now.

What I Fixed:
✅ Resolved the merge conflict by combining the best of both branches:

Kept the enhanced features (badge + CTA buttons) from feat branch
Added dark mode support from main branch
✅ Updated all elements for dark mode compatibility:

Badge: dark:bg-teal-800 dark:text-teal-100
Heading: text-black dark:text-gray-100
Description: text-black dark:text-gray-400
Primary button: dark:bg-teal-500 dark:hover:bg-teal-600
Secondary button: dark:bg-gray-800 dark:text-teal-400 dark:border-teal-400 dark:hover:bg-gray-700

Prachiikhar25 committed:
Enhanced Features

Prachiikhar25 committed:
Merge branch 'feat' of https://github.com/Prachiikhar25/fitness-chatbot into feat

Prachiikhar25 commented:
I have also added the changes for the issue #4

please review it 



Enhanced Features:
✅ Smart Context Management - The bot now provides follow-up responses based on previous topics ✅ Cross-Topic Integration - Responses intelligently reference related topics ✅ 8 Interactive Sample Buttons - Color-coded for easy topic identification:

🟢 Teal: Original topics (workouts, nutrition, sleep, motivation)
🟣 Purple: Yoga/Flexibility
🔵 Blue: Hydration
🟠 Orange: Rest/Recovery
🟢 Green: Muscle Recovery
✅ Professional Response Quality - Each response includes actionable advice and guided next steps ✅ Maintained Code Structure - All additions follow the existing project patterns and formatting

The chatbot now covers a comprehensive range of fitness topics while maintaining the project's professional quality and user experience!
